lib/bezier.js

```diff
@@ -795,6 +795,12 @@
           np3 = this.get(e);
 
           arc = utils.getccenter(np1, np2, np3);
+          
+          //also save the t values
+          arc.t_s = s;
```

Comment:
I'd recommend using normal JS objects for this:
```
arc.t = {
  s: s,
  m: m.
  e: e
};
```
That said, can you give a use case for these values? Because having the triplet `{s,m,e}` does not seem very useful to me, and I'd like to understand what you'd use them for.